networkparams_test.go

```diff
@@ -0,0 +1,74 @@
+// Copyright (c) 2016 The btcsuite developers
+// Copyright (c) 2016 The Decred developers
+// Use of this source code is governed by an ISC
+// license that can be found in the LICENSE file.
+
+package main
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/decred/dcrd/blockchain"
+	"github.com/decred/dcrd/chaincfg"
+)
+
+//TestDecredNetworkSettings checks Network-specific settings
```

Comment:
Should have a space after the `//` and end with a period to be consistent with the rest of the code.